docs/decision-records/2023-07-06-committer-retirement/README.md

```diff
@@ -0,0 +1,22 @@
+# Adding guidelines for Committers retirement
+
+## Decision
+[Committer of the EDC](https://projects.eclipse.org/projects/technology.edc/who) project have a higher activity requirement, as they directly impact the ability of others to contribute. Therefore, *Committers* with an extended period away from the project with no activity will be retired from their role of *Committer* in the EDC and will be required to go through the meritocratic process again after re-familiarizing themselves with the current state.
+
+Inactive *Committers* are defined as members of the EDC *Committers* with no contributions across any repository within 6 months (according to EF’s suggestion), or recognizable non-code contributions to the project. *Committers* are required to show up in *Committer-Meetings*.
```

Comment:
If we say committers are required to show up to meetings, we should define what that is. I also don't think it should be an onerous requirement. Maybe we should rephrase this as:

 "Committers are expected to regularly attend commuter meetings"